rfd/0097-teleport-connect-usage-metrics.md

```diff
@@ -0,0 +1,175 @@
+will be supplied with UUID generated by Connect with `connect.` prefix. The value will be created on the start and
+stored in a file in the app data directory, so it will not change between restarts.
+
+> **NOTE:** As stated above, Connect events are tied to the application instance (or just the client machine). It means
+> that PostHog's _Person_ for Connect and for cluster will be a different thing. They should not be merged.
+
+### User agreement
+
+On the start, Teleport Connect will ask user to opt in to volunteer anonymized metrics and usage-data with standard
+message "Are you OK sending anonymized usage data about Teleport Connect? This will help us to improve product".
+
+If the user refuses, Connect will not send any usage data.
+
+### How will collecting metrics support product development?
+
+In the initial version, it should help with getting answers to the following questions:
+
+#### How many unique users download and use Teleport Connect today?
+
+To answer the first part of the question, download counts from `goteleport.com/download` are needed. These will be
+collected from access logs from CloudFront CDN.
+
+To calculate how many users use Teleport Connect on a daily basis, a metric like DAU (Daily Active Users) can be used.
+This metric can be based on a specific event, but in this case it should be calculated using any event. For example,
+user logged only once in a given day to refresh certs for a DB proxy connection - such user can be considered as active.
+
+#### What features are the most popular?
+
+Usage of each feature will be measured basing on events from the [events](#events) section. They will allow to generate
+various statistics, like the most common kinds of connections or just show the usage of particular features like Access
+Requests.
+
+#### What platforms are the most popular?
+
+This will be measured in two ways.
+
+- Based on downloads count for each platform.
+- Based on a real usage - every event will contain the OS field. These events will be then aggregated by unique users.
+
+#### How usage grows or shrinks over time?
+
+PostHog allows to create [Trends](https://posthog.com/manual/trends) basing on DAU. It can be used to show how usage
+changes in a given period of time.
+
+## Events
+
+### `connect.cluster.login`
+
+Successful login to a cluster.
+
+Event properties:
+
+- `cluster_name`: string (anonymized)
+- `user_name`: string (anonymized)
+- `connector_type`: string
+- `os`: string (set once on a user properties)
+- `arch`: string (set once on a user properties) - CPU architecture
+- `os_version`: string (set on a user properties)
+- `connect_version`: string (set on a user properties)
+- `distinct_id`: string
+
+### `connect.protcol.run`
+
+Connecting to the protocol.
+
+Event properties:
+
+- `cluster_name`: string (anonymized)
+- `user_name`: string (anonymized)
+- `protocol`: one of `ssh`/`proxy_db`/`kube`
+- `distinct_id`: string
+
+### `connect.accessRequests.create`
```

Comment:
For consistency, let's use the singular form. We did `connect.protocol.run`, not `connect.protocols.run`, so let's do the same here.

```suggestion
### `connect.accessRequest.create`
```